DriveTrain.java

```diff
@@ -190,6 +197,29 @@ private double calculateTurn(double distanceLeft, double distanceRight){
         return turn;
     }
 
+    // Return average of array.
```

Comment:
Can this be moved to a utility class?  We may need to smooth other values and it'd be nice to leverage the same code elsewhere.